src/cis/fam_cis_direct.cpp

```diff
@@ -1145,22 +1145,19 @@ void Fam_CIS_Direct::wait_for_copy(void *waitObj) {
 
 void *Fam_CIS_Direct::backup(uint64_t srcRegionId, uint64_t srcOffset,
                              uint64_t srcMemoryServerId, string BackupName,
-                             uint32_t uid, uint32_t gid, uint64_t size) {
+                             uint32_t uid, uint32_t gid, uint64_t size, mode_t mode, string dataitemName) {
```

Comment:
It may not secure to pass the mode from client to memory server. mode needs to be taken from metadata service

Reply:
should we allow user to pass the permission mode of the backup? 
Dataitem permission might have some restricted mode, which user does not want for its backup?
As for this implementation, we can say it the backup permission mode will be same as that of corresponding dataitem.